service/src/main/kotlin/fi/espoo/evaka/messaging/bulletin/BulletinQueries.kt

```diff
@@ -163,42 +186,148 @@ fun Database.Read.getSentBulletinsByUnit(
 ): Paged<Bulletin> {
     // language=sql
     val sql = """
-        SELECT COUNT(b.*) OVER (), b.*, br.group_id AS group_id, dg.name AS group_name, concat(e.first_name, ' ', e.last_name) AS created_by_employee_name
-        FROM bulletin b
+        WITH limited_bulletins AS (
```

Comment:
not quite sure if this will now paginate correctly.. I think not?

Reply:
I mean it looks like it's now paginating a set of bulletins but then filtering some out afterwards, which means that pages will have varying amounts of items and some may even be empty.